ios/push-notifications.md

```diff
@@ -14,47 +14,40 @@ title: Push Notifications
 Enable your users to receive mobile push messages sent from the Apple (APNs) and Google (FCM/GCM) platforms. The CLI deploys your push notification backend using [Amazon Pinpoint](http://docs.aws.amazon.com/pinpoint/latest/developerguide/).
 You can also create Amazon Pinpoint campaigns that tie user behavior to push or other forms of messaging.
 
-## Set Up Your Backend
-
-1. Complete the [Get Started](./start) steps before you proceed.
-
-2. Complete the [Setting Up APNS Guide](./push-notifications-setup-apns). This will guide you through the process of setting up an App ID, SSL certificate, provisioning profile, entitlements, distribution certificate, and provisioning profile, which are required to deliver notifications to iOS devices.
-
-3. Use the CLI to add notifications to your cloud-enabled backend and app.
+## Pre-requisite
 
-    In a terminal window, navigate to your project folder (the folder that typically contains your project level `xcodeproj` file), and add the SDK to your app.
+- Enrolled in the Apple Developer Program
+- Actual device to run the app on
 
-    ```bash
-    $ cd ./YOUR_PROJECT_FOLDER
-    $ amplify add notifications
-    ```
+1. Complete the [Get Started](./start) steps before you proceed.
 
-4. Set up your backend to support receiving push notifications:
+2. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
 
-    - Choose Apple Push Notification Service (APNs).
+3. Plug in the device and make sure it is registered and there are no errors on this page provisioning the profile.
 
-        ```
-        > APNS
-        ```
+4. On this page, click on `+ Capability`, and add `Push Notification` and `Background Modes`. For `Background Modes`, have `Remote notifications` checked.
 
-    - Choose Certificate as your authentication method.
+## Set Up Your Backend
 
-        ```
-        > Certificate
-        ```
+1. Add analytics to your app and allow guests and unauthenticated users send analytics events.
+```
+amplify add analytics
+```
 
-    - Provide the path to your P12 certificate that you created in [Setting Up iOS Push Notifications](./push-notifications-setup-apns).
+2. Provision the backend
+```
+amplify push
+```
 
-   Use the steps in the next section to connect your app to your backend.
+3. Drag `awsconfiguration.json` into your project if haven't done so already from a previous set up, with `copy as needed` checked.
```

Comment:
its actually already there for me now based on following the getting started steps! didn't even have to drag it in :)

Reply:
haha because of amplify-tools?

Reply:
did some rewording around amplify-tools and move this point down to the next section